util/archive_handler.go

```diff
@@ -52,10 +54,28 @@ func GetMtaIDFromArchive(mtaArchveFilePath string) (string, error) {
 }
 
 func readZipFile(file *zip.File) ([]byte, error) {
+	err := validateFileSize(file, defaultMaxDescriptorSize)
+	if err != nil {
+		return nil, err
+	}
+
 	reader, err := file.Open()
 	if err != nil {
 		return nil, err
 	}
 	defer reader.Close()
 	return ioutil.ReadAll(reader)
 }
+
+func validateFileSize(file *zip.File, maxFileSize int64) error{
+	fileInfo := file.FileInfo()
+
+	if fileInfo.Size() > maxFileSize {
+		return fmt.Errorf("The size %d of file %s exceeds max size limit %d", fileInfo.Size(), file.Name, defaultMaxDescriptorSize)
+	}
+	return nil
+}
+
+func setDefaultMaxDescriptorSize(maxFileSize int64){
```

Comment:
This is not the go style to configure some variable. Use DefaultMaxDescriptorSize instead of defaultMaxDescriptorSize  in this way it will be public and it could be configured.